mlfinlab/labeling/fixed_time_horizon.py

```diff
@@ -0,0 +1,60 @@
+"""
+Chapter 3.2 Fixed-Time Horizon Method, in Advances in Financial Machine Learning, by M. L. de Prado
+
+Work "Classification-based Financial Markets Prediction using Deep Neural Networks" by Dixon et al. (2016) describes how
```

Comment:
Your line length is a little bit too long. 

Please can you make it a bit smaller so that all your code will fit on the review without me having to scroll left and right.